widget_driver/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# CHANGELOG
+
+## 0.0.6
+
+* Fixes issues with ignoring of state changes
+* using new generated mixin from `widget_driver_generator ^0.0.6`
```

Comment:
Is "enable" then a good phrase? Because I'd like to reference the generator here, because the driver is in and on itself not dependant on the generator. However if the versions don't match up, the generator code throws errors or the widget_driver doesn't take advantage of the generated code.
Thats why I would say the widget_driver enables the newly generated code by the generator.